.github/workflows/test-all.yml

```diff
@@ -0,0 +1,69 @@
+name: Test browsers
+
+on:
+    pull_request:
+        branches:
+            - master
+            - stable
+            - major
+    push:
+        tags:
+            - v*
+        branches:
+            - master
+            - stable
+            - major
+
+jobs:
+    # Runs tests for selected Angular versions and all environments
+    # Top-level Angular version (as per package-lock.json) will be used for PRs. All versions of Angular will be tested otherwise.
+    test-all:
+        runs-on: ubuntu-20.04
+        name: Run tests
+        strategy:
+            matrix:
+                browser: [BrowserStack_Edge, BrowserStack_Safari, BrowserStack_IE11, Chrome, Firefox]
```

Comment:
Maybe we should consider running all browsers via BS? It's already done for React v2. Local browsers should be used for local development only.